app/frontend/src/components/TextField.tsx

```diff
@@ -37,7 +36,6 @@ export default function TextField({
     <MuiTextField
       {...otherProps}
       variant={variant}
-      id={id}
```

Comment:
Why was this removed? We need this previously so [MUI correctly links label up with the right ID](https://material-ui.com/components/text-fields/#accessibility). Removing it will break this

Reply:
I've also removed  `id` from the destructured props. So `otherProps` will still have the `id` prop. The spread operator used on line 38 will nicely pass all these props, including the `id` prop, to the MuiTextField component.

So this change is just a 'code style' issue.